.github/workflows/ci-nightly.yaml

```diff
@@ -1,13 +1,13 @@
 name: Kata Containers Nightly CI
 on:
   schedule:
-    - cron: '0 0 * * *'
+    - cron: '0 */2 * * *'
 
 jobs:
   kata-containers-ci-on-push:
     uses: ./.github/workflows/ci.yaml
     with:
       commit-hash: ${{ github.sha }}
-      pr-number: ${{ github.sha }}
+      pr-number: "nightly"
```

Comment:
This is fine, but I guess the alternative if we want to track more clear tracking of the commit tested would be to do a `cut -c1-8` or something to get the 'short' sha out

Reply:
Great, thanks for clarifying